Light Vox Engine/Input/InputManager.cpp

```diff
@@ -0,0 +1,18 @@
+#include "InputManager.h"
+
+using namespace Input;
+
+void InputManager::OnMouseDown( WPARAM buttonState, int x, int y )
+{
+
+}
+
+void InputManager::OnMouseUp( WPARAM buttonState, int x, int y )
```

Comment:
Hmm should we typedef the WPARAM to make it more cross-platform? I know only PCs have mouse but it just makes things less complicated later on